src/renderer/ui/components/settings/ToggleableOption.js

```diff
@@ -1,23 +1,35 @@
 import React, { Component, PropTypes } from 'react';
+import muiThemeable from 'material-ui/styles/muiThemeable';
 import Checkbox from 'material-ui/Checkbox';
 
 import SettingsProvider from '../generic/SettingsProvider';
 
-class SettingsCheckbox extends Component {
+const SettingsCheckbox = muiThemeable()(class extends Component {
```

Comment:
cc / the linter doesn't like this.

It would look better as

```js
class SettingsCheckbox extends Component {
  ..
}

const ThemedSettingsCheckbox = muiThemeable()(SettingsCheckbox);
```

And then just use `ThemedSettingsCheckbox`